project/Dependencies.scala

```diff
@@ -83,6 +83,7 @@ object Dependencies {
 
   val DynamoDB = Seq(
     libraryDependencies ++= Seq(
+      "org.scala-lang.modules" %% "scala-java8-compat" % "0.8.0", // BSD-style
```

Comment:
Dependency to java8-compat comes transitively from akka-actor. So it is not needed to include it here as well.